common/utils/src/main/resources/error/error-classes.json

```diff
@@ -2750,6 +2750,12 @@
     ],
     "sqlState" : "42000"
   },
+  "NULL_INDEX" : {
+    "message" : [
+      "Value at index <index> is null."
```

Comment:
```suggestion
      "Row value at the position <position> is NULL."
```

Reply:
Do you want me to change this when this wasn't the original message? I think the ticket just wants me to classify the error, not necessarily introduce user facing message changes

Reply:
I changed it and made a note in the summary that we also changed the message